config/docker/cards-publication.yml

```diff
@@ -1,8 +1,20 @@
 
 # uncomment kafka.consumer.group-id to enable Kafka
-#  kafka:
-#    consumer:
-#      group-id: opfab-command
+spring:
+  deserializer:
+    value:
+      delegate:
+        #        class: io.confluent.kafka.serializers.KafkaAvroDeserializer
+        class: org.opfab.cards.publication.kafka.consumer.KafkaAvroWithoutRegistryDeserializer
+  serializer:
+    value:
+      delegate:
+        #        class: io.confluent.kafka.serializers.KafkaAvroSerializer
+        class: org.opfab.cards.publication.kafka.producer.KafkaAvroWithoutRegistrySerializer
```

Comment:
would it be possible to change it directly it the docker defaut config in services/cards-publication/src/main/resources/application.yml ?